components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Registry/src/Services/PublisherRegistry.cs

```diff
@@ -133,31 +133,26 @@ public sealed class PublisherRegistry : IPublisherRegistry, IPublisherEndpointQu
                         if (patched.Configuration == null) {
                             patched.Configuration = new PublisherConfigModel();
                         }
-                        if (request.Configuration.JobOrchestratorUrl != null) {
-                            patched.Configuration.JobOrchestratorUrl =
-                                string.IsNullOrEmpty(
-                                    request.Configuration.JobOrchestratorUrl.Trim()) ?
-                                        null : request.Configuration.JobOrchestratorUrl;
-                        }
-                        if (request.Configuration.HeartbeatInterval != null) {
-                            patched.Configuration.HeartbeatInterval =
-                                request.Configuration.HeartbeatInterval;
-                        }
-                        if (request.Configuration.JobCheckInterval != null) {
-                            patched.Configuration.JobCheckInterval =
-                                request.Configuration.JobCheckInterval.Value.Ticks == 0 ?
-                                    null : request.Configuration.JobCheckInterval;
-                        }
-                        if (request.Configuration.MaxWorkers != null) {
-                            patched.Configuration.MaxWorkers =
-                                request.Configuration.MaxWorkers <= 0 ?
-                                    null : request.Configuration.MaxWorkers;
-                        }
-                        if (request.Configuration.Capabilities != null) {
-                            patched.Configuration.Capabilities =
-                                request.Configuration.Capabilities.Count == 0 ?
-                                    null : request.Configuration.Capabilities;
-                        }
+                        
+                        patched.Configuration.JobOrchestratorUrl =
```

Comment:
@dacolgit, will this differentiate between the cases where `JobCheckInterval` is explicitly set to `null` in the request body and where it is omitted ? Maybe this is related to serialization/deserialization of request body.

So, in the case of this body, I am assuming that `JobCheckInterval` will be set to `null`, right ?

```json
{
  "configuration": {
    "JobCheckInterval": null
  }
}
```

But in the case of this body, where `JobCheckInterval` is not present, `JobCheckInterval` will not be modified, right ?

```json
{
  "configuration": {
    "HeartbeatInterval": "00:02:00"
  }
}
```

Reply:
yes